<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>602</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>894</width>
    <height>602</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>894</width>
    <height>602</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BeatSync</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="subscribeDetail">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>291</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="subscribeName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:30px;
color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.831056, y2:0.347, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(51, 51, 51, 255));
font-weight:900;
</string>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="subscribeDesc">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:15px;
color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.831056, y2:0.347, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(51, 51, 51, 255));</string>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QLabel" name="subscribeAuthor">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight:800;
font-size:18px;
color:qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));</string>
     </property>
     <property name="text">
      <string>Author</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="downloadList">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>260</y>
      <width>291</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:3px;
font-size:11px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testsBtn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>测试项目</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="topBarBtn" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="subscribes">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>581</width>
      <height>581</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateCacheBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>更新本地缓存</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="topBarBtn" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dListTips">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>570</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:800;
color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(100, 100, 100, 255), stop:1 rgba(58, 58, 58, 255));</string>
    </property>
    <property name="text">
     <string>待下载歌曲</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="topBarBtn" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dCtrlTip">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>570</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:800;
color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(100, 100, 100, 255), stop:1 rgba(58, 58, 58, 255));</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
